pkg/routeagent_driver/handlers/mtu/mtuhandler.go

```diff
@@ -51,6 +52,7 @@ const (
 
 type mtuHandler struct {
 	event.HandlerBase
+	ipFamily         k8snet.IPFamily
 	localClusterCidr []string
 	pFilter          packetfilter.Interface
 	tableType        packetfilter.TableType
```

Comment:
Maybe we should have 2 fields in mtuHandler struct for LocalCIDRIPSet and RemoteCIDRIPSet ? for IPv4 values should be constants.LocalCIDRIPSet and 	constants.RemoteCIDRIPSet (	RemoteCIDRIPSet       = "SUBMARINER-REMOTECIDRS"
	LocalCIDRIPSet        = "SUBMARINER-LOCALCIDRS"
) and we can append "-6" for IPv6 family